xurls_test.go

```diff
@@ -272,6 +350,10 @@ func TestRegexes(t *testing.T) {
 		{`3ffe:2a00:100:7031::1`, nil},
 		{`test.foo.com:8080/path`, nil},
 		{`foo@bar.com`, nil},
+
+		// An IP address in URI host position must be bracketed unless it is IPv4.
+		// https://www.rfc-editor.org/rfc/rfc3986#section-3.2.2
+		//{`http://1080::8:800:200c:417a/path`, nil},
```

Comment:
should this be commented out?

Reply:
I'd love to uncomment it, but the current code still incorrectly accepts bare IPv6 addresses as IRI hostname components, and I haven't yet found a good way to fix that.

Reply:
fair; perhaps add a TODO to clarify

Reply:
Of course; done.